README.md

```diff
@@ -3,741 +3,470 @@
 
-struct BatchClaimWithWitness {
-    // ... same first 5 parameters as all claims
-    bytes32 witness; // Hash of the witness data.
-    string witnessTypestring; // Witness typestring appended to existing typestring.
-    BatchClaimComponent[] claims; // IDs and amounts.
-    address claimant; // The claim recipient; specified by the arbiter.
-}
+**1. Create a Resource Lock (Deposit Tokens):**
+    - A sponsor starts by depositing assets (native tokens or ERC20s) into The Compact. This action creates ERC6909 tokens representing ownership of the resource lock.
+    - During deposit, the sponsor defines the lock's properties: the **allocator** (who must be registered first, see [Allocators (Infrastructure)](#allocators-infrastructure) and [Key Concepts: Allocators](#allocators)), the **scope** (single-chain or multichain), and the **reset period** (for forced withdrawals and emissary replacements). These are packed into a `bytes12 lockTag`. A resource lock's ID is a combination of its lock tag and the underlying token's address.
+    - Deposit methods (see [`ITheCompact.sol`](./src/interfaces/ITheCompact.sol)):
+        - Native tokens: [`depositNative`](./src/interfaces/ITheCompact.sol#L81)
+        - ERC20 tokens (requires direct approval): [`depositERC20`](./src/interfaces/ITheCompact.sol#L97)
+        - Batch deposits (native + ERC20): [`batchDeposit`](./src/interfaces/ITheCompact.sol#L114)
+        - Via Permit2 (optionally gasless): [`depositERC20ViaPermit2`](./src/interfaces/ITheCompact.sol#L131), [`batchDepositViaPermit2`](./src/interfaces/ITheCompact.sol#L157)
+    - See [Key Concepts: Resource Locks](#resource-locks) for details on token handling.
 
-struct QualifiedBatchClaimWithWitness {
-    // ... same first 5 parameters as all claims
-    bytes32 witness; // Hash of the witness data.
-    string witnessTypestring; // Witness typestring appended to existing typestring.
-    bytes32 qualificationTypehash; // Typehash of the qualification payload.
-    bytes qualificationPayload; // Data used to derive qualification hash.
-    BatchClaimComponent[] claims; // IDs and amounts.
-    address claimant; // The claim recipient; specified by the arbiter.
-}
+**2. Create a Compact:**
+    - To make locked funds available for claiming, a sponsor creates a compact, defining terms and designating an **arbiter**.
+    - **Option A: Signing an EIP-712 Payload:** The sponsor signs a `Compact`, `BatchCompact`, or `MultichainCompact` payload. This signed payload is given to the arbiter. See [Key Concepts: Compacts & EIP-712 Payloads](#compacts--eip-712-payloads).
+    - **Option B: Registering the Compact:** The sponsor (or a trusted third party) registers the *hash* of the intended compact details using [`register`](./src/interfaces/ITheCompact.sol#L204) or combined deposit-and-register functions. See [Key Concepts: Registration](#registration).
```

Comment:
Doesn't have to be a trusted third party; if anything that implies that you're able to delegate ability to register (which you currently aren't able to do).

I think we just clarify that any third party can convert a signature that's currently valid into a registration, and that any party that explicitly supplies deposited tokens can register a compact spending those tokens on behalf of any sponsor they indicate.